mods/d2k/rules/aircraft.yaml

```diff
@@ -1,4 +1,4 @@
-carryall.scripted:
+carryall.reinforce:
```

Comment:
Actually they should all be targetable. The initial idea of making carryalls untargetable was questionable, because [they were targetable in the original](http://media.moddb.com/images/games/1/14/13377/big-battle.jpg).  (or was it their cargo that was targetable, I'm not sure now).
Anyway, I'll remove `TargetableAircraft` from all carryalls for now, but it probably should be brought back when the proper balancing is being done.